executor/errors.go

```diff
@@ -64,6 +64,7 @@ var (
 	ErrNotSupportedWithSem           = dbterror.ClassOptimizer.NewStd(mysql.ErrNotSupportedWithSem)
 	ErrPluginIsNotLoaded             = dbterror.ClassExecutor.NewStd(mysql.ErrPluginIsNotLoaded)
 	ErrSetPasswordAuthPlugin         = dbterror.ClassExecutor.NewStd(mysql.ErrSetPasswordAuthPlugin)
+	ErrFuncNotEnabled                = dbterror.ClassExecutor.NewStdErr(mysql.ErrNotSupportedYet, parser_mysql.Message("%-.32s works only when %-.32s is set in config file", nil))
```

Comment:
The warning is not clear that the behavior is a noop. This distinction is important in understanding the risks. May I suggest instead:

> LOCK TABLE is not supported. To enable this experimental feature, set 'enable-table-lock' in the configuration file.

Otherwise LGTM.

Reply:
Yes, it's too confusing. I think we should return warning whether `noop_funcs` enabled or not. Maybe controlled by 'enable-table-lock' is enough. @morgo

Reply:
Modified according to your suggestion. @morgo

@AilinKid PTAL